raiwidgets/tests/test_cohort.py

```diff
@@ -542,14 +548,21 @@ def test_cohort_serialization_include_exclude_methods(self, method):
                                          column='age')
         cohort_int = Cohort(name="Cohort New Int")
         cohort_int.add_cohort_filter(cohort_filter_int)
-        json_str = json.dumps(cohort_filter_int,
-                              default=cohort_filter_json_converter)
+
+        json_str = cohort_int.to_json()
         assert method in json_str
         assert '1' in json_str
         assert '2' in json_str
         assert '3' in json_str
         assert 'age' in json_str
 
+        cohort_int_new = Cohort.from_json(json_str)
+        assert cohort_int_new.name == cohort_int.name
```

Comment:
Suggestion: Can these be refactored into a helper method to validate cohorts are equal?  Then this code and the code above could be in the method call.  It might be useful when adding more tests in the future.

Reply:
Added an __eq__ in Cohort class that returns True in case the cohort and its underlying filters are equal.